.github/workflows/bwc-tests.yml

```diff
@@ -0,0 +1,42 @@
+name: Backwards Compability Suite
+
+on: [workflow_dispatch]
+
+jobs:
+  last-supported-major:
+    name: Make sure that the last supported major version can mov
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 11
```

Comment:
As you know, you may run into issues with using jdk 11. Do we want/need to account for other jdk versions? What happens when we move from a version that supports jdk 17/18 to one that does not?

Reply:
I wrote these tests as specific one off- I imagine this file might become a much bigger matrix of all testing we want done, but wanted to start smaller, I'll create a follow up issue.  

- https://github.com/opensearch-project/security/issues/2260

Reply:
Makes perfect sense to me!